package.json

```diff
@@ -37,10 +37,11 @@
   },
   "devDependencies": {
     "@aws-sdk/types": "^3.341.0",
-    "@dvsa/eslint-config-ts": "^3.0.0",
     "@dvsa/cvs-type-definitions": "^2.0.19",
+    "@dvsa/eslint-config-ts": "^3.0.0",
     "@types/aws-lambda": "^8.10.114",
     "@types/jest": "^28.1.8",
+    "@types/lodash": "^4.14.195",
```

Comment:
This is needed because we need to create deep clones of the imported test data in order to avoid repeating a lot of boiler plate code in each test. If you don't deep clone them and import the test data it causes random failures in the tests depending on the order in which they execute.